support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -2,106 +2,81 @@ package com.gu.emailservices
 
 import com.gu.emailservices.SubscriptionEmailFieldHelpers._
 import com.gu.i18n.Currency
-import com.gu.salesforce.Salesforce.SfContactId
 import com.gu.support.promotions.Promotion
 import com.gu.support.workers._
 import com.gu.support.workers.states.PaymentMethodWithSchedule
 
-// Output Json should look like this:
-//
-// {
-//  "To": {
-//    "Address": "gow5hckeqzc86qqx4rc@gu.com",
-//    "SubscriberKey": "gow5hckeqzc86qqx4rc@gu.com",
-//    "ContactAttributes": {
-//      "SubscriberAttributes": {
-//        "ZuoraSubscriberId": "A-S00045678",
-//        "Date of first payment": "3 August 2018",
-//        "Address 2": "",
-//        "Trial period": "14",
-//        "First Name": "first_GOW5hckeqZC86QQx4Rc",
-//        "Last Name": "last_GOW5hckeqZC86QQx4Rc",
-//        "Country": "United Kingdom"
-//        "Default payment method": "Credit/Debit Card",
-//        "Currency": "£",
-//        "Post Code": "N1 9AG",
-//        "Subscription term": "month",
-//        "City": "London",
-//        "Subscription details": "£11.99 every month",
-//        "EmailAddress": "gow5hckeqzc86qqx4rc@gu.com",
-//        "Payment amount": "11.99",
-//        "Address 1": "Kings Place"
-//      }
-//    }
-//  },
-//  "DataExtensionName": "digipack"
-//}
+object DigitalPackEmailFields {
 
-case class DigitalPackEmailFields(
-  paidSubPaymentData: Option[PaymentMethodWithSchedule]
-) extends SubscriptionEmailFields {
+  def build(
+    subscriptionEmailFields: SubscriptionEmailFields,
```

Comment:
all the `apply` repetition in the old file is collapsed into this one case class